tests/example.at

```diff
@@ -118,3 +118,5 @@ POSITIVE_TEST([tree],[example])
 POSITIVE_TEST([triangle],[example])
 POSITIVE_TEST([turing1],[example])
 POSITIVE_TEST([weighted_distances],[example])
+POSITIVE_TEST([json],[example])
```

Comment:
Move the tests from example/example.at to semantic/semantic.at, so that the new json feature is tested in standard tests.